binscripts/rvm-installer

```diff
@@ -268,7 +272,7 @@ fetch_versions()
   _repo=$3
   case ${_domain} in
     (bitbucket.org)
-      _url=https://${_domain}/api/1.0/repositories/${_account}/${_repo}/branches-tags
+      _url="https://api.${_domain}/2.0/repositories/${_account}/${_repo}/refs/tags?sort=-name&pagelen=20"
```

Comment:
FYI I added `&pagelen=20` to increase from the default page size of 10.  We could go as high as fetching the 100, but I thought that might be excessive (unless someone really wants to install a version from long ago).  But it any case, it is not possible to go and fetch _all_ tags in one go, they would have to be accumulated over multiple requests.



> pagelen: Current number of objects on the existing page. Globally, the minimum length is 10 and the maximum is 100. Some APIs may specify a different default.

https://developer.atlassian.com/bitbucket/api/2/reference/meta/pagination

Reply:
I believe 20 is enough as it goes a few years back :) We do not release too often.